dropbox_rclone/dropbox_rclone/scripts/cs_dropbox_download.py

```diff
@@ -197,6 +210,7 @@ def download(
             continue
 
         print(f"### Extracting {t}")
+        tar = tarfile.open(t, "r")
```

Comment:
Do you use this?

Reply:
Doesn't look like it, seems to be something I missed when having to do a merge conflict.